src/layout/components/Page/index.vue

```diff
@@ -127,9 +126,8 @@ export default {
 
   .page-content {
     height: calc(100% - 20px);
-    //Todo)) 注释掉的原因：当页面动态高度变化时就会导致页面最下方的按钮被 hidden 掉
-    //overflow-x: hidden;
-    //overflow-y: auto !important;
+    overflow-x: hidden;
+    overflow-y: auto !important;
 
     ::v-deep > div {
       margin-bottom: 50px;
```

Comment:
None. The provided code doesn't contain any irregularities, potential issues, or optimization suggestions that I could identify. It's well-documented with the correct scss file being used which is a good coding practice, and it's also correctly formatted according to SCSS guidelines.

To summarize: The provided code snippet is fine and does not require improvement based on the given information. There aren't any known problems in this specific section of the document/codebase.
 
Note: In general, there isn't typically an issue discussing current day vs cutoff dates when asking questions here as these tools generally have access to all versions over time but may vary slightly depending upon the context of the conversation within a broader user knowledge base. Please let me know if you need help specifically around certain areas!